tap4j/src/test/java/org/tap4j/parser/TestConstants.java

```diff
@@ -29,15 +29,16 @@
 import java.lang.reflect.InvocationTargetException;
 
 import org.junit.Test;
+import org.tap4j.model.Patterns;
 
 /**
- * Tests for parser Constants.
+ * Tests for parser Patterns.
  */
 public class TestConstants {
 
     @Test
     public void testUtilConstructor() throws IllegalArgumentException, InstantiationException, IllegalAccessException, InvocationTargetException {
-        final Constructor<?> c = Constants.class
+        final Constructor<?> c = Patterns.class
```

Comment:
Name makes sense. Though will probably make the next release a major version (I'll take care of updating the since tag in javadoc).

The TAP Plug-in loads tap4j classes, and changes like this may break it as it relies on XStream for XML de/serialization (though I think the plug-in is not serializing the Patterns class, more likely to be serializing classes from the model package).